scripts/run_e2e_tests.py

```diff
@@ -260,8 +258,6 @@ def setup_and_install_dependencies(skip_install):
     """Run the setup and installation scripts."""
     if not skip_install:
         install_third_party_libs.main()
-    setup.main(args=[])
```

Comment:
Will it work for a fresh repo? Considering setup creates basic structure for storing thord_party and tools.

Also, now after these changes how do we set chrome_bin for these tests?

Reply:
`setup.main` is called from `install_third_party_libs.main`

Reply:
Ah, I see thanks for the info!